space-member.ts

```diff
@@ -0,0 +1,15 @@
+import { Kind_V2, Signer, SpaceMember } from "./nostr.ts";
+import { PublicKey } from "./key.ts";
+import { nowTime } from "./_helper.ts";
+
+export function prepareAddMember(author: Signer, member: string): Error | Promise<SpaceMember> {
+    if (PublicKey.FromString(member) instanceof Error) {
+        return new Error("The member's public key is incorrect.");
+    }
+    return author.signEventV2({
+        pubkey: author.publicKey.hex,
+        kind: Kind_V2.SpaceMember,
```

Comment:
1. rename to `AddSpaceMember`
2. Are we finally settle down our name to `Space`? If so, we need to rename all concepts to "space"

Reply:
1. It seems that we will not have `EditSpaceMember` or `DeleteSpaceMember`. Deletion can be done using the generic `Delete` Event. So I think it is more appropriate to use `SpaceMember` directly.

2. Recently, we have all been using `Space` as a term of reference, but it may change after the new product name is finalized.